scanapi/template_render.py

```diff
@@ -18,3 +19,13 @@ def _loader(is_external):
         return FileSystemLoader(searchpath="./")
 
     return PackageLoader("scanapi", "templates")
+
+
+def render_body(request):
+    """ Render body according to its request content type """
+    content_type = request.headers.get("Content-Type")
+    body = request.body
+    if content_type in ["application/json", "text/plain"]:
+        return body.decode("UTF-8")
```

Comment:
since python 3 not necessary set utf-8 in decode

Reply:
Yep, decode will use the default string encoding

Reply:
Thanks !